eng/common/scripts/common.ps1

```diff
@@ -0,0 +1,27 @@
+$global:RepoRoot = Resolve-Path "${PSScriptRoot}..\..\..\.."
+$global:EngDir = Join-Path $global:RepoRoot "eng"
+$global:EngCommonDir = Join-Path $global:EngDir "common"
+$global:EngCommonScriptsDir = Join-Path $global:EngCommonDir "scripts"
+$global:EngScriptsDir = Join-Path $global:EngDir "scripts"
+
+# Import required scripts
+. (Join-Path $global:EngCommonScriptsDir SemVer.ps1)
+. (Join-Path $global:EngCommonScriptsDir Changelog-Operations.ps1)
+. (Join-Path $global:EngCommonScriptsDir Package-Properties.ps1)
+
+# Setting expected from common languages settings
+$global:Language = "Unknown"
+$global:PackageRepository = "Unknown"
+$global:packagePattern = "Unknown"
+$global:MetadataUri = "Unknown"
+
+# Import common language settings
+$EngScriptsLanguageSettings = Join-path $global:EngScriptsDir "Language-Settings.ps1"
+if (Test-Path $EngScriptsLanguageSettings) {
+  . $EngScriptsLanguageSettings
+}
+
+# Transformed Functions
+$ExtractPkgProps = "Extract-${Language}-PkgProperties"
```

Comment:
In isolation I couldn't tell you what this function is supposed to do and isn't clear what is different between this and the next function. We should consider a slightly better name. Perhaps `Parse-${Langauge}-PackageInfoFromRepo`, or something along those lines. 

Although Parse doesn't look to be an approved PS verb https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7 so perhaps we use "Get".

How about:
- `Get-${Language}-PackageInfoFromRepo`
- `Get-${Language}-PackageInfoFromPackageFile`
- `Publish-${Language}-GithubIODocs`